go-controller/pkg/ovn/master.go

```diff
@@ -371,6 +371,19 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 			return err
 		}
 
+		// Create a cluter-wide port group to deny all multicast traffic.
```

Comment:
I think it is probably more efficient to just add a multicast_allow_router portgroup+acl with the ovn_cluster_router ports in it. Then we don't need a a port group with all the ports in it.

Reply:
I think you're right.  I'll have to try it out first though.  There's no restriction in OVN wrt adding ls ports connected to a router to a port group but I don't think I ever saw this being used in practice so I'm not sure if there are caveats.

Reply:
@trozet It works fine, thanks for the suggestion.  I just pushed an updated version.